packages/rds/initial-bootstrap.ts

```diff
@@ -0,0 +1,170 @@
+import axios from "axios";
+import { createConnection, Connection } from "typeorm";
+import { CloudFormationCustomResourceEvent, CloudFormationCustomResourceResponse } from "aws-lambda";
+
+import { getDatabaseCredentials } from "./auth/secrets";
+import path from "path";
+import * as fs from "fs";
+
+type User = {
+  name: string;
+  dbPrivileges: string[];
+  tablePrivileges: string[];
+};
+
+type Config = {
+  databaseName: string;
+  secretName: string;
+  databaseHost: string;
+  caBundleFile: string;
+};
+
+const USERS: User[] = [
+  {
+    name: "atat_api_read",
```

Comment:
If the use is effectively a role, it'd be fine to leave it as is.

Reply:
That's basically what it is. I'll define an `enum` with context/comment in `infrastructure/lib/constructs/database.ts` but to parameterize it in this function becomes a little too complex (this can no longer be `const`).

Reply:
The addition of `DatabaseUsers` enum helps.